GOVERNANCE.md

```diff
@@ -253,10 +255,9 @@ A TSC member has significant sway in software design decisions. For this reason,
 
 #### Nomination
 
-- To be nominated, a nominee is expected to already be active in technical discussions and performing some of the responsibilities of a TSC member.
-- You can be nominated by any existing Core member (L3 or above). Note: This includes all existing TSC members as well.
-- Once nominated, there will be a vote by existing Core members.
-- See [vote rules & requirements](#voting) for info on how this vote works.
+- The makeup of the TSC is decided by the project Steward.
+- A core member may self-nominate by sending a private message to the Steward. The Steward shall provide time and effort to discuss the nomination with any nominee, and provide feedback that would be helpful to the nominee as they evaluate their own nomination.
```

Comment:
is there a better way to word this? Is it worth having this? What I'm trying to capture here is that anyone feels like they can say "hey, I want to be on the TSC" and the Project Steward has a responsibility to engage with them in evaluating their self-nomination, and communicating back to them constructive feedback based on the Steward's evaluation of them as a nominee.

Reply:
Sounds clear enough to me